src/app/features/tech-record/components/tech-record-history/tech-record-history.component.ts

```diff
@@ -48,4 +49,8 @@ export class TechRecordHistoryComponent {
         return `/tech-records/${this.vehicle.systemNumber}/`;
     }
   }
+
+  updateTechRecordInEdit(index: number): void {
+    this.techRecordService.updateEditingTechRecord(this.vehicle.techRecord[this.pageStart || 0 + index]);
```

Comment:
Should this function not be called with `true` as a second param to make sure we update rather than patch? The function is confusing but it seems like the second param makes it so we dispatch a fresh version of the selected tech record, not merge with the current editing version.

Reply:
Suggestion applied.